templates/.gitignore

```diff
@@ -1,2 +0,0 @@
-node_modules
```

Comment:
It's funny that the template `.gitignore` was ignored itself and thus we had this bug 😆